tests/test_suites/ShortSystemTests/test_auto_recovery_repair.sh

```diff
@@ -13,6 +13,10 @@ mkdir dir
 saunafs setgoal 3 dir
 FILE_SIZE=$(( 4 * SAUNAFS_CHUNK_SIZE )) file-generate dir/file
 
+if is_windows_system; then
+	sleep 5
```

Comment:
These tests were doing some IO operation of over 1mb. It seems that under WSL1 the CS are running very slow and there are a few of them in each of these cases. Thus, to succeed on the following checks (like checking if there is some amount of chunks of this or that type) done in those tests we must wait for the CS. It applies to several other tests.

I chose 5 seconds cuz it was the least time it was passing everywhere, I'll add as you said a function called ```wait_chunkservers``` which just sleeps 5s. 

Also note that I'm not interested in checking the speed or well-behavior of the CSs, and I suppose them to be perfect.

Reply:
Again, the testing feels kinda faulty because it's not really testing the actual environment where SaunaFS is being used.

But please add the reasoning as comments there, for future developers.